services/resource-usage-tracker/tests/unit/with_dbs/test_api_resource_tracker_pricing_plans.py

```diff
@@ -170,10 +217,31 @@ async def test_get_default_pricing_plan_for_service(
     assert data["pricing_units"][1]["unit_name"] == "M"
     assert data["pricing_units"][2]["unit_name"] == "L"
 
-    _PRICING_UNIT_ID = 2
     url = URL(f"/v1/pricing-plans/{_PRICING_PLAN_ID}/pricing-units/{_PRICING_UNIT_ID}")
     response = await async_client.get(f'{url.with_query({"product_name": "osparc"})}')
     assert response.status_code == status.HTTP_200_OK
     data = response.json()
     assert data
     assert data["pricing_unit_id"] == _PRICING_UNIT_ID
+
+    url = URL(f"/v1/services/{_SERVICE_KEY_2}/{_SERVICE_VERSION_2}/pricing-plan")
+    response = await async_client.get(f'{url.with_query({"product_name": "osparc"})}')
+    assert response.status_code == status.HTTP_200_OK
+    data = response.json()
+    assert data
+    assert len(data["pricing_units"]) == 1
+    assert data["pricing_units"][0]["unit_name"] == "XXL"
+
+    bigger_version = "3.10.5"
+    url = URL(f"/v1/services/{_SERVICE_KEY_2}/{bigger_version}/pricing-plan")
+    response = await async_client.get(f'{url.with_query({"product_name": "osparc"})}')
```

Comment:
TIP: since you only need path, you can always do ` await  async_client.get(url.with_query(...).path)`